brew_view/__init__.py

```diff
@@ -120,6 +126,7 @@ def _setup_tornado_app():
         (r'{0}config/swagger/?'.format(prefix), SwaggerConfigHandler),
         (r'{0}version/?'.format(prefix), VersionHandler),
         (r'{0}api/v1/spec/?'.format(prefix), SpecHandler),
+        (r'{0}events/?'.format(prefix), EventSocket),
```

Comment:
Good call - my initial thought is to put it with the versioned ones. Let me know if you'd rather have it in beta.

Reply:
Ahh, I remember why I punted on this decision :smile: 

We already have a POST `api/vbeta/events/` route that allows for publishing your own events. I think we eventually want to support retrieving events by an ID (assuming you've been saving them to mongo)?

Now I'm thinking it would be better to have this be something besides `events` so we don't muddy the RESTfulness.

Reply:
And now I remember more ... the apispec package we're using to construct the swagger spec doesn't like that we're putting a websocket handler in our 'published' path list. Which makes sense - it's not really designed for non-REST things. See:

https://github.com/swagger-api/swagger-socket/issues/47
https://github.com/OAI/OpenAPI-Specification/issues/55

So far all of the API we've wanted to publish has fit nicely into REST, so it's fit nicely into swagger (and apispec). We'll have to decide how to move forward with documenting the websocket support.